src/rendering/RenderUtil.cc

```diff
@@ -321,8 +321,44 @@ class ignition::gazebo::RenderUtilPrivate
   /// \brief A map of entity id to thermal camera sensor configuration
   /// properties. The elements in the tuple are:
   /// <resolution, temperature range (min, max)>
-  public:std::unordered_map<Entity,
+  public: std::unordered_map<Entity,
       std::tuple<double, components::TemperatureRangeInfo>> thermalCameraData;
+
+  /// \brief A helper function that removes the sensor associated with an
+  /// entity, if an associated sensor exists. This should be called in
+  /// RenderUtil::Update.
+  /// \param[in] _entity The entity that should be checked for an associated
+  /// sensor.
+  public: void RemoveSensor(const Entity _entity);
+
+  /// \brief A helper function that removes the bounding box associated with an
+  /// entity, if an associated bounding box exists. This should be called in
+  /// RenderUtil::Update.
+  /// \param[in] _entity The entity that should be checked for an associated
+  /// bounding box.
+  public: void RemoveBoundingBox(const Entity _entity);
+
+  /// \brief A helper function for updating lights. This should be called in
+  /// RenderUtil::Update.
+  /// \param[in] _entityLights A map of entity IDs to their light updates.
+  public: void UpdateLights(
+              const std::unordered_map<Entity, msgs::Light> &_entityLights);
+
+  /// \brief A helper function for updating the thermal camera. This should be
+  /// called in RenderUtil::Update.
+  /// \param[in] _thermalCamData The thermal camera data that needs to be
+  /// updated.
```

Comment:
I added a note below this line that references `RenderUtilPrivate::thermalCameraData`: https://github.com/ignitionrobotics/ign-gazebo/blob/334bca682f46b2139497b22f74adea11c4b37412/src/rendering/RenderUtil.cc#L351

The thought I had was that it'd be better to reference the variable which has this information documented instead of repeating the documentation again here. That way, if the variable is ever modified, we don't need to worry about updating documentation in multiple places. Is this approach okay, or would it still be good to add documentation here?

Reply:
ok yeah sounds good